### PR TITLE
Minor additions

### PR DESCRIPTION
**Pull request by erithwik**
I just added some minor changes: 
1) Added an option to see what courses a person has taken (as that seemed to be missing in the `get_profile` function. 
2) Added some helper methods (writing and appending to a file) that could make development and general usage easier for some people.

### CODE REVIEW

File: linkedin_api/linkedin.py [hunk 1]
```diff
@@ -418,6 +418,12 @@ def get_profile(self, public_id=None, urn_id=None):
             del item["entityUrn"]
         profile["honors"] = honors
 
+        # massage [courses] data
```
Comment (tomquirk) [inline linkedin_api/linkedin.py:421]: **praise**: well done adding this!

File: linkedin_api/utils/helpers.py [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
 def get_id_from_urn(urn):
     """
     Return the ID of a given Linkedin URN.
 
     Example: urn:li:fs_miniProfile:<id>
     """
     return urn.split(":")[3]
+
+
+def write_to_file(input_data, file_name):
```
Comment (tomquirk) [inline linkedin_api/utils/helpers.py:13]: **issue**: I understand that these would be helpful, but I feel like these are general enough that they should not live in this library - they don't directly support the goals of the project.

Could you please remove them?

Comment (erithwik) [reply linkedin_api/utils/helpers.py]: Sure, I removed them and have committed.

**Comment by erithwik** [comment]
I have removed the unnecessary functions in utils.py as you have mentioned.

File: linkedin_api/utils/helpers.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
+import json
```
Comment (tomquirk) [inline linkedin_api/utils/helpers.py:1]: Looks like this import isn't required anymore: can it be removed?

**Comment by tomquirk** [comment]
Bump!

File: linkedin_api/utils/helpers.py (hunk 3 above)
Comment (tomquirk) [reply linkedin_api/utils/helpers.py:1]:
```suggestion
```

**Review by tomquirk — APPROVED** [review]

**Pull request closed without merging**